pkg/util/filesystem/util_windows.go

```diff
@@ -49,7 +49,7 @@ func IsUnixDomainSocket(filePath string) (bool, error) {
 	// Due to the absence of golang support for os.ModeSocket in Windows (https://github.com/golang/go/issues/33357)
 	// we need to dial the file and check if we receive an error to determine if a file is Unix Domain Socket file.
 
-	// Note that querrying for the Reparse Points (https://docs.microsoft.com/en-us/windows/win32/fileio/reparse-points)
+	// Note that querying for the Reparse Points (https://docs.microsoft.com/en-us/windows/win32/fileio/reparse-points)
```

Comment:
i cannot approve the pkg change but if you remove it we can merge a PR that fixes typos in cmd/kubeadm.
also please try finding more typos in cmd/kubeadm.

try keeping the commits squashed to one!